### DISCUSSION TITLE
reassembling starred repos

### DISCUSSION THREAD

**Discussion by lodwicz** [community/community#43152] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello! I thought that after classifying starred repos into lists, the repos would disappear from the general list and appear only under the lists, leaving only unclassified repos; otherwise, I don't know their use if not to unclutter the interface.

**Comment by radar-blip** [comment]
It's just a way to organize your repositories by putting them into categories.

**Comment by cycl0ne** [comment]
I tried today also list and yes, this would be a cool way. I was also astonished that they "didnt" disappear. Clicking on the list did the job.

So in my opinion, we are missing a "Starred but not assigned to a list" option/filter. I have over the years about 200 stars. i categorized now 2 lists. 60 and 1 .. and now im fiddling to find the missing 139 Stars.

And while we are at it, maybe it is too much, but what about lists in lists? ;-) Lets say you have starred the following Categories 3dprintingmodels, 3dprintingtools, 3dprintingmods, 3dprintinganything. so i could be cool to have an overall: 3dprinting list with the categories: models, tools, mods, other.
